packages/playground/windows/playground/MainPage.xaml

```diff
@@ -1,13 +1,145 @@
+            <RowDefinition Height="*"/>
+        </Grid.RowDefinitions>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="auto"/>
+            <ColumnDefinition Width="*"/>
+        </Grid.ColumnDefinitions>
 
-  <Grid x:Name="x_PaneContainer">
-    <Grid.ColumnDefinitions/>
-  </Grid>
+        <AppBarButton
+            x:Name="x_LoadButton"
+            Label="Load"
+            Icon="Play"
+            Style="{StaticResource AppBarButtonRevealStyle}"
+            VerticalAlignment="Center"
+            Grid.Column="0"
+            Grid.RowSpan="2"
+            Click="OnLoadClick"
+            Margin="4"
+        />
+
+        <Grid
+            Grid.Row="0"
+            Grid.Column="1"
+            BorderThickness="1,0,0,0"
+            BorderBrush="{StaticResource SystemBaseMediumColor}">
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition Width="auto"/>
+                <ColumnDefinition Width="*"/>
+                <ColumnDefinition Width="auto"/>
+                <ColumnDefinition Width="auto"/>
+            </Grid.ColumnDefinitions>
+
+            <TextBlock
+                Grid.Column="0"
+                Margin="8,2,2,2"
+                VerticalAlignment="Center"
+                Text="JavaScript File (no extension):"/>
+            <ComboBox x:Name="x_entryPointCombo"
+                Grid.Column="1"
+                Margin="2,2,2,2"
+                MinWidth="250"
+                SelectedIndex="0"
+                VerticalAlignment="Center"
+                HorizontalAlignment="Stretch"
+                SelectionChanged="x_entryPointCombo_SelectionChanged">
+                <ComboBox.Items>
+                    <ComboBoxItem Content="Samples\rntester" />
+                    <ComboBoxItem Content="Samples\accessible" />
+                    <ComboBoxItem Content="Samples\animation" />
+                    <ComboBoxItem Content="Samples\calculator" />
+                    <ComboBoxItem Content="Samples\callbackTest" />
+                    <ComboBoxItem Content="Samples\click" />
+                    <ComboBoxItem Content="Samples\control" />
+                    <ComboBoxItem Content="Samples\flexbox" />
+                    <ComboBoxItem Content="Samples\focusTest" />
+                    <ComboBoxItem Content="Samples\geosample" />
+                    <ComboBoxItem Content="Samples\image" />
+                    <ComboBoxItem Content="Samples\index" />
+                    <ComboBoxItem Content="Samples\messages" />
+                    <ComboBoxItem Content="Samples\mouse" />
+                    <ComboBoxItem Content="Samples\scrollViewSnapSample" />
+                    <ComboBoxItem Content="Samples\simple" />
+                    <ComboBoxItem Content="Samples\text" />
+                    <ComboBoxItem Content="Samples\textinput" />
+                    <ComboBoxItem Content="Samples\ticTacToe" />
+                    <ComboBoxItem Content="Samples\view" />
```

Comment:
can these be dynamicaly discovered?

Reply:
I'm going to punt on this for this PR.  The old playground had these hard coded too...